docs/install.md

```diff
@@ -0,0 +1,30 @@
+# Install CVNBot
+
+## Installation
+
+1. Compile the code and build the executable for `Release` configuration (not `Debug`),
+   by running the following command:
+
+   `xbuild src/CVNBot.sln /p:Configuration=Release`
```

Comment:
In [travis](https://travis-ci.com/countervandalism/CVNBot/jobs/160153504#L1552) Line 1552.
`>>>> xbuild tool is deprecated and will be removed in future updates, use msbuild instead <<<<`
We still use xbuild ?

Reply:
I am a beginner with C-sharp and Mono. I only know xbuild because that's what we used originally. Also, I think in the past the Mono project did not have msbuild. But I see that with the current version (Mono 5+), it also has msbuild.

I propose that we switch to msbuild, but maybe in a separate change, so that we can test that and also to update `infrastructure` steps.

Reply:
Yeah, Do it in futurely :)

Reply:
Done, see https://github.com/countervandalism/CVNBot/pull/53.